flink-end-to-end-tests/flink-end-to-end-tests-sql/pom.xml

```diff
@@ -199,4 +199,25 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <!-- This profile adds dependencies needed to execute the tests with Hadoop 3 -->
+        <profile>
+            <id>hadoop3-tests</id>
+            <dependencies>
+                <dependency>
+                    <groupId>org.apache.hadoop</groupId>
+                    <artifactId>hadoop-hdfs-client</artifactId>
+                    <version>${flink.hadoop.version}</version>
+                    <scope>test</scope>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>com.fasterxml.jackson.core</groupId>
+                            <artifactId>jackson-annotations</artifactId>
```

Comment:
Why is this being excluded?

Reply:
I print mvn dependency in hadoop2.8.5, found jackson-annotations is not included in hadoop-hdfs-client, but it existed in hadoop3, so I exclude it to keep consistency.

Reply:
I'm not sure if that's a good idea. At the end of the day hadoop 3 has a different dependency tree, and trying to keep the close may just cause problems in the future when something suddenly stops working.

Reply:
OK, I remove it.